### DISCUSSION TITLE
Interested in testing out subnets?

### DISCUSSION THREAD

**Discussion by elenananana** [hirosystems/stacks-hyperchains#97] [General]
We are looking for teams interested in testing out hyperchains and providing feedback. If you are interested, please leave a comment in this discussion with details regarding your project, use case and contact information.

**Comment by Jamil** [comment]
We'd love to test out hyperchains and start building out infrastructure for it at https://gamma.io, an NFT marketplace on Stacks. It'd be great to use hyperchains for the marketplace functionality, as well as for new mints if possible.

Contact info: jamil@gamma.io

**Comment by fiftyeightandeight** [comment]
+1 for ALEX (https://www.alexgo.io). We would like to first test out DEX functionalities.

Thank you.

**Reply by elenananana** [reply]
Hello - could you please share a good email or contact info? thanks!

**Comment by MarvinJanssen** [comment]
The Clarity Lab is also interested in trying out hyperchains. We do not have an explicit project or use case in mind but would be interested in exploring connectors to bridge SIP009, SIP010, SIP013, and more back and forth.

cc @radicleart

**Comment by philiphacks** [comment]
Would like to try this with Arkadiko (https://arkadiko.finance/, USDA stablecoin). Thanks!

philip@arkadiko.finance

**Comment by kjchiang3** [comment]
We'd love to test out and try building tools with the hyperchain.
We can test functionalities for virtually all of our operations to make sure our DEX and related tools can help other Stacks startups take advantage of our platform features.

(communication@stackswap.org)

**Comment by FriendsFerdinand** [comment]
Interested in hyperchains for the Zest Protocol (https://www.zestprotocol.com/) and interacting with the bitcoin blockchain.

email: friendsferdinand@protonmail.com

**Comment by pseudozach** [comment]
We'd love to join as https://lnswap.org and test how native swaps can be done with hyperchains.
Thanks,
zach@trustmachines.co

**Comment by aulneau** [comment]
I'd love to experiment with writing some indexing software on top of hyperchains. I've been working on new options for building extensible apis for the base layer stacks chain, and would love to extend the work towards hyperchains. 

-> hi@fungible.systems

**Comment by dantrevino** [comment]
Boom is also interested in testing from a marketplace and nft management perspective.  [dantrevino@gmail.com](mailto:dantrevino@gmail.com)

**Comment by ryanwaits** [comment]
+1 for StackerDAOs - we would love to play around with hyperchains for our voting functionality (and potentially other areas where it might make sense).

> ryan@stackerdaos.com

**Comment by john-aigora** [comment]
Please count NeoSwap in.  Hyperchains will make an enormous difference to us as we have to get many people to sign the same contract, and the current high latency is a major challenge that we have to overcome to get our multi-way trades approved by all involved.

**Reply by JakeBlockchain** [reply]
Second this.

**Comment by hyeongjun-dev** [comment]
We would love to try building functionalities with Hyperchain. - https://nfpstudio.io
Thanks!

leo@despread.io

**Comment by whoabuddy** [comment]
+1 on behalf of CityCoins, I think this would be a great avenue to explore in quite a few capacities:

- mining and stacking operations for a city
- city-related applications that require high throughput
- wrapped asset exposure for liquidity

Curious to see what this could unlock!

jason@joinfreehold.com

**Comment by mardi00** [comment]
+1 I want to try to develop small "games" using $ICE from StacksPops using hyperchains

axopoa@protonmail.com

**Comment by garyriger** [comment]
We'd love to test out HyperChains at HeyLayer. We have so many fun products coming that will help with NFT creation and utility tools. We have the Need for Speed. 

Contact:
gary@heylayer.com

**Comment by bogachevanton** [comment]
We will be happy to participate in Hyperchains testing.  Our product is 90 STX Entertainment Platform (https://90stx.xyz/), an entertainment platform with currently launched raffle functionality. There are also daily mini-raffles with NFTs from different collections, which are presented in the Stacks network. Now it is planned to launch an instant game and new types of raffles, which will require fast transactions.

Contact:
anton@apeonthehill.xyz
Telegram:
https://t.me/bogachev_anton

**Comment by obsidianbtc** [comment]
+1 for https://byzantion.xyz/. Also would like to get started adding to our indexing framework.

**Comment by igorsyl** [comment]
Interested here as test user as well as running a miner HC node.
igor@trustmachines.co

**Comment by yknl** [comment]
Xverse wallet is definitely interested in supporting Hyperchains from the wallet side.

**Comment by KhawarR** [comment]
We would love to explore Hyperchains, especially for the gaming side. We are also developing some tools for stacks and we will be exploring more on that front too.

Contact Info: khawar@tintash.com
Discord for our first collection: http://discord.gg/VapingApeClub
Website: tintash.com

**Comment by danieldunne** [comment]
HireVibes would love to participate in Subnet testing. We'd also like to be a Subnet (HC) miner. 

Contact info: daniel@hirevibes.io
https://www.hirevibes.com/

**Comment by abdelghafour** [comment]
Deep Lake Derivatives (https://twitter.com/DeepLakeFinance) would love to test its derivatives on top of hyperchains.

Contact info: abdel@deeplake.finance
App: https://app.deeplake.finance/

Thank you.

**Comment by corbo369** [comment]
Hello, I am from the Fungible Flip team, building on Stacks and currently in the Web3 Startup Lab.

Hyperchains are critical to our success and we would love to be able to begin testing to integrate them.

Contact Info:
https://twitter.com/FungibleFlip
fungibleflip@gmail.com

**Comment by Gordon1770** [comment]
Hi Elenananana 

Stacks Australia are working with Griffith University Intern and PHD student Dora Zelenyanszki to develop project-based learning programs for Postgrad Students. The idea of creating a subnet for their university could be one option, it could also attract Devs and PHD researchers to work on and add to the Subnets project and offer excellent opportunities their participants to build applications for the subnets. If there is an opportunity for Dora to participate in some way on the development program as it is happening that would be great experience for her to best develop the programs for their university. Dora will post and intro to herself here too. Many thanks. Gordon Chapter lead at Stacks Australia

**Reply by elenananana** [reply]
love this! we will follow up soon. :)

**Reply by elenananana** [reply]
what is the best contact info/ email for you?

**Comment by AAMFSJR** [comment]
Hello,

I am the CEO of eruSwap a DEX being built upon Stacks.

We would love to go fully on-chain but would need access to subnets to create a better user experience.

Subnets would allow us to increase transparency to our users through our smart contracts, and self-custody of their assets. While keeping the experience quick, which is needed in DEX.

Hyperchains are critical to our success and we would love to be able to begin testing to integrate them.

A little bit more about us can be found here:
https://www.eruswap.com/
https://twitter.com/EruSwap

You can reach us at fabricealomo@gmail.com

Thanks!

**Comment by danieldunne** [comment]
Hey, great to hear back on this! Yes our best contact is ***@***.***
- we look forward to hearing from you!

*Best regards,*
Daniel

Co-Founder & CEO
HireVibes Services
Calendly <https://calendly.com/daniel-hirevibes/15min> -
Telegram: @DanDunne




HireVibes.com <http://HireVibes.io>

HireVibes users are making *impact*! Check out our ReFi Donations
<https://www.hirevibes.io/donations> 🌴

**Comment by elenananana** [comment]
Hi everyone! Please register for the subnets call scheduled for next Mon 5/1. Cheers

https://lu.ma/uxk6zi1s

**Reply by igorsyl** [reply]
> Hi everyone! Please register for the subnets call scheduled for next Mon 5/1. Cheers
> 
> https://lu.ma/uxk6zi1s

Thank you for sharing @elenananana . I'm evaluating Subnets as a deployment platform for sBTC, so please keep in me the loop. I signed up for the meeting.

**Comment by r0zar** [comment]
Hello :wave: 

I'm part of the new cohort of the Bitcoin Startup Lab.
I'm using subnets in my project. 

email: ross@pointblankdev.com

**Reply by igorsyl** [reply]
Congratulations!

**Comment by Rapha-btc** [comment]
**Team of 2 backend + 1 frontend engineers seeking 1-2 federation mining partners to build bonding curve DEXes. Proven product-market fit: HYPE hit $13B, Odin Fun exceeded Liquidium's yearly volume in weeks.**

**Product lines (hacked only by 1 fe fyi)**
- [Faktory.Fun](https://faktory.fun/) (AI DAOs)
- [fak.fun](https://fak.fun/) (memes)

**Ready to go all-in. Please DM [@raphastacks](https://x.com/RaphaStacks) on X.**
